react/src/pages/RoutingListPage.tsx

```diff
@@ -113,7 +113,26 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
             name
             status
             endpoint_id
-            image
```

Comment:
To ensure backward compatibility, we should use different images based on the version.

Reply:
I just added conditional statement to check whether current core version supports `image_object` key in endpoint.

Reply:
Please add  `@deprecatedSince` like this:
`image @deprecatedSince(version: "23.09.9")`